README.md

```diff
@@ -159,6 +159,25 @@ Kubernetes users:\
 
 Most plugins should be supported out-of-the-box, or maybe require some minimal changes. See this [dashboard](https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=18341) for known compatibility issues.
 
+## Compatability with Jenkins > 2.199
```

Comment:
I don't think this huge blob of text belongs in the main README,
I would move it to a sub page and reference it here:
https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/docs

Reply:
Those docs are not discoverable - so I disagree.
Can be moved by somewhen when the docs are fixed :-)

ref: https://plugins.jenkins.io/configuration-as-code

Reply:
@oleg-nenashev any idea why that's happening? we released 7 days ago and the url looks right

Reply:
created https://github.com/jenkinsci/configuration-as-code-plugin/issues/1206